src/remodal.js

```diff
@@ -431,6 +422,22 @@
   }
 
   /**
+   * Generates a string separated by dashes and prefixed with NAMESPACE
+   * @private
+   * @param str, str, ...
+   * @returns str
+   */
+  function namespaced() {
+    var result = NAMESPACE;
+
+    for (var i = 0; i < arguments.length; ++i) {
```

Comment:
Not so critical in this case, but: `for (var i = 0, len = arguments.length; i < len; ++i)`